plugin-src/gitVisualization/boxHighlighting.ts

```diff
@@ -102,31 +103,32 @@ async function removeCurrentHighlighting(box: Box): Promise<void> {
     await renderManager.remove(`${box.header.getId()}Inner-gitDiff`)
 }
 
-async function addShowChangesButton(box: FileBox, changedFile: ChangedFile): Promise<void> {
+async function addOpenChangesButton(box: FileBox, changedFile: ChangedFile): Promise<void> {
+    const buttonId: string = `${box.getId()}-openChangesButton`
     await renderManager.addElementTo(`${box.header.getId()}Inner`, {
         type: 'button',
-        children: 'Show Changes',
-        id: box.getId() + '-openChangesButton',
+        children: 'Open Changes',
+        id: buttonId,
         title: 'Uses git difftool: https://git-scm.com/docs/git-difftool\\n' +
             'Make sure you have it configured, e.g. for VSCode:\\n' +
             '[diff]\\n' +
             '    tool = default-difftool\\n' +
             '[difftool \\"default-difftool\\"]\\n' +
             '    cmd = code --wait --diff \\$LOCAL \\$REMOTE',
-        style: {float: 'right', marginRight: '5px', marginTop: '2px'},
+        style: {float: 'right', marginRight: '5px', marginTop: '2px', cursor: 'pointer'},
         onclick: () => {
-            showChanges(changedFile)
+            openChanges(changedFile)
         }
     })
+    await renderManager.addEventListenerTo(buttonId, 'mousedown', () => {
+    })
```

Comment:
add a comment why these weird lines are there and that it is a workaround
```suggestion
    await renderManager.addEventListenerTo(buttonId, 'mousedown', () => {}) // workaround to catch events before dragManager starts dragging
```